File: hw/blacio/u1/hw08ObjectSave/JS/Tax.js
class Tax {
    static computeIva(ivaPercentage, amount) {
        let total;
        total = amount * ivaPercentage / 100;
        return total;
    }

    static computeIce(icePercentage, amount) {
        let total;
        total = amount * icePercentage / 100;
        return total;
    }
}

module.exports = Tax;

File: ws/arroyo/u1/ws14ReferencedDT/Taxes/src/ec/edu/espe/sales/model/Tax.java

package ec.edu.espe.sales.model;

/**
 *
 *  @author Alfonso Arroyo <As-Byte Wizards> DDCC ESPE
 */
public class Tax {
    
    public static float computeIce(float icePercentage, float amount){
        float total;
        total = amount * icePercentage / 100;
        return total;
    }
    public static float computeIva(float ivaPercentage, float amount){
        float total;
        total = amount*ivaPercentage/100;
        return total;
    }
    
    
}

